commons/src/main/java/org/infinispan/commons/marshall/StreamingMarshaller.java

```diff
@@ -124,4 +128,92 @@
 
    void start();
 
+   static StreamingMarshaller from(StreamAwareMarshaller marshaller) {
+      return new StreamingMarshaller() {
+         @Override
+         public ObjectOutput startObjectOutput(OutputStream os, boolean isReentrant, int estimatedSize) throws IOException {
+            throw new UnsupportedOperationException();
```

Comment:
@wburns In this case the alternative to throwing an exception is to return null, I'm not sure that is any better as it will result in NPE from existing code.

Reply:
Weird, I must have been looking at a different diff or something. Since this requires invoking the new `from` method you can disregard this comment.